load_issues.py

```diff
@@ -0,0 +1,178 @@
+from pathlib import Path
+from typing import Any, Dict, List
+import os
+import textwrap
+
+import typer
+import yaml
+
+from github import Github
+from github.Issue import Issue
+from github.Repository import Repository
+
+from loguru import logger
+
+DEFAULT_CONFIG = {"owner": "uw-ssec", "repo": "codeuw", "repos": []}
+
+
+def load_config(config_file: str = ".codeuw-config.yml") -> Dict[str, Any]:
+    """
+    Load configuration file
+
+    Parameters
+    ----------
+    config_file : str, optional
+        The string path to configuration file,
+        by default ".codeuw-config.yml"
+
+    Returns
+    -------
+    Dict[str, Any]
+        The configuration dictionary
+    """
+    config_path = Path(config_file)
+    config = yaml.safe_load(config_path.read_text())
+
+    # Set defaults, in case they're not in the config file
+    for key, value in DEFAULT_CONFIG.items():
+        config.setdefault(key, value)
+
+    return config
+
+
+def setup_github() -> Github:
+    """
+    Setup Github API
+
+    Returns
+    -------
+    Github
+        Github API object
+    """
+    from github import Auth
+
+    # using an access token
+    # it'll look for environment variable GITHUB_TOKEN
+    auth = Auth.Token(os.environ.get("GITHUB_TOKEN"))
+
+    # Public Web Github
+    return Github(auth=auth)
+
+
+def parse_issue_template(source_repo: Repository) -> Dict[str, Any]:
+    """
+    Parse the issue template from the source repository
+    and output a dictionary of the template
+
+    Parameters
+    ----------
+    source_repo : Repository
+        The source repository to parse the issue template from
+
+    Returns
+    -------
+    dict
+        The dictionary of the issue template
+    """
+    issue_template = source_repo.get_contents(path=".github/ISSUE_TEMPLATE/task.yml")
```

Comment:
It's a bit confusing to have `issue_template` and `task_template`. How about `issue_template_text` and `issue_template`?